blocks/suggest/suggest.bemhtml.js

```diff
@@ -3,7 +3,12 @@ block('suggest')(
         return applyNext({ _suggest : this.ctx });
     }),
 
-    js()(true),
+    js()(function() {
+        var prevJs = applyNext(),
```

Comment:
it's enough just
```js
return applyNext() || true;
```

Reply:
But if I use 
```js
js()(function() {
        return applyNext() || true;
})
```
js-parameters don't added to data-bem content for suggest block

Reply:
@d3m1d0v are you sure? https://goo.gl/5cLptw

Reply:
@belozer Yes, and  when suggest focused it not initializing (no added `suggest_js_inited` class)

Reply:
@d3m1d0v please show your html. In demo all ok.